packages/artifact/docs/artifact-contributions.md

```diff
@@ -0,0 +1,30 @@
+# Contributions
```

Comment:
Could we rename this to CONTRIBUTIONS.md and put it at `packages/artifact` so that it's easier to find?

CONTRIBUTIONS.md is based on guidelines from GitHub https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors